commit 22c9d52bc03b880045ab1081890a38f11b272ae7
Author: Jeff Layton <jlayton@redhat.com>
Date:   Thu Apr 16 13:48:49 2009 -0400

    cifs: remove unneeded bcc_ptr update in CIFSTCon
    
    This pointer isn't used again after this point. It's also not updated in
    the ascii case, so there's no need to update it here.
    
    Pointed-out-by: Dave Kleikamp <shaggy@linux.vnet.ibm.com>
    Signed-off-by: Jeff Layton <jlayton@redhat.com>
    Signed-off-by: Steve French <sfrench@us.ibm.com>

Backported to Debian's 2.6.24 by dann frazier <dannf@debian.org>

diff -urpN linux-source-2.6.24.orig/fs/cifs/connect.c linux-source-2.6.24/fs/cifs/connect.c
--- linux-source-2.6.24.orig/fs/cifs/connect.c	2009-04-29 00:59:27.000000000 -0600
+++ linux-source-2.6.24/fs/cifs/connect.c	2009-04-29 01:00:34.000000000 -0600
@@ -3427,7 +3427,6 @@ CIFSTCon(unsigned int xid, struct cifsSe
 						tcon->nativeFileSystem,
 						(__le16 *) bcc_ptr,
 						length, nls_codepage);
-				bcc_ptr += (2 * length) + 2;
 			}
 			/* else do not bother copying these information fields*/
 		} else {
